File: 3110-score-of-a-string/3110-score-of-a-string.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        
        l = 0
        i = 1
        res = 0

        while(i < len(s)):
          res+= abs((ord(s[l]) - ord(s[i])))
          print(s[l])
          print(s[i])
          l+=1
          i+=1
          
        
        return res
          

File: 3110-score-of-a-string/3110-score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
        
      int l = 0;
      int r = 1;
      int res = 0;
      while(r < s.length()){
        
        res+= Math.abs(((int) s.charAt(l)) - ((int) s.charAt(r)));
        l++;
        r++;
        
      }
      return res;
    }
}